diff --git a/Configurations/unix-Makefile.tmpl b/Configurations/unix-Makefile.tmpl
--- a/Configurations/unix-Makefile.tmpl	2023-04-15 09:03:11.013850300 -0700
+++ b/Configurations/unix-Makefile.tmpl	2023-04-15 09:42:03.926810800 -0700
@@ -1380,13 +1380,13 @@
 $(BLDDIR)/util/opensslwrap.sh: Makefile
 	@if [ "$(SRCDIR)" != "$(BLDDIR)" ]; then \
 	    mkdir -p "$(BLDDIR)/util"; \
-	    ln -sf "../$(SRCDIR)/util/`basename "$@"`" "$(BLDDIR)/util"; \
+	    ln -sf "../$(SRCDIR)/util/`basename "$@"`" "$(BLDDIR)/util/"; \
 	fi
 
 $(BLDDIR)/apps/openssl.cnf: Makefile
 	@if [ "$(SRCDIR)" != "$(BLDDIR)" ]; then \
 	    mkdir -p "$(BLDDIR)/apps"; \
-	    ln -sf "../$(SRCDIR)/apps/`basename "$@"`" "$(BLDDIR)/apps"; \
+	    ln -sf "../$(SRCDIR)/apps/`basename "$@"`" "$(BLDDIR)/apps/"; \
 	fi
 
 FORCE:
